coffea/lumi_tools/lumi_tools.py

```diff
@@ -123,18 +117,19 @@ def __call__(self, runs, lumis):
                 An array of dtype `bool` where valid (run, lumi) tuples
                 will have their corresponding entry set ``True``.
         """
+        # fill numba typed dict
+        _masks = Dict.empty(key_type=types.uint32, value_type=types.uint32[:])
+        for k, v in self._masks.items():
+            _masks[k] = v
```

Comment:
I was at first a bit concerned that we re-build the numba typed dict on every evaluation (i.e. every chunk) but I decided to see how fast it was:
```
from coffea.lumi_tools import LumiMask
lumimask = LumiMask(
    "tests/samples/Cert_294927-306462_13TeV_EOY2017ReReco_Collisions17_JSON.txt"
)
from numba import types
from numba.typed import Dict
def test(m):
    _masks = Dict.empty(key_type=types.uint32, value_type=types.uint32[:])
    for k, v in m.items():
        _masks[k] = v

%timeit test(lumimask._masks)
```
and it only takes `822 µs ± 9.66 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)` so this is fine for a once-per-chunk operation. What I would have done is write `__getstate__` and `__setstate__` to convert the masks to numba dictionaries during the unpickling, but it doesn't make too much difference I guess.

Reply:
Good point, I usually try to solve pickling things first by restructuring a bit and use `__getstate__` and `__setstate__` as a last rescue. 
Good test, that the performance is not impacted! :)
`numba.typed.Dict` is a _experimental_ feature according to the Numba API, so maybe at some point this [should be/gets] on its own pickable in the future.